The failures on Linux are like this:

```
CMake Error at /mnt/vcpkg-ci/installed/x64-linux/share/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake:129 (message):
  '/mnt/vcpkg-ci/packages/bcg729_x64-linux/debug/share/bcg729' does not
  exist.
Call Stack (most recent call first):
  ports/bcg729/portfile.cmake:21 (vcpkg_cmake_config_fixup)
  scripts/ports.cmake:142 (include)
```

--- a/ports/bcg729/portfile.cmake
+++ b/ports/bcg729/portfile.cmake
@@ -9,15 +9,16 @@ vcpkg_from_github(
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" ENABLE_STATIC)
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" ENABLE_SHARED)
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
-    PREFER_NINJA
     OPTIONS
         -DENABLE_STATIC=${ENABLE_STATIC}
         -DENABLE_SHARED=${ENABLE_SHARED}
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(PACKAGE_NAME Bcg729)
 
 file(INSTALL "${SOURCE_PATH}/LICENSE.txt" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
 

--- a/ports/bcg729/vcpkg.json
+++ b/ports/bcg729/vcpkg.json
@@ -1,7 +1,17 @@
 {
   "name": "bcg729",
-  "version-string": "1.1.1",
-  "port-version": 1,
+  "version": "1.1.1",
+  "port-version": 2,
   "description": "Bcg729 is an open source implementation of the ITU G.729 Annex A and B codec.",
-  "homepage": "https://github.com/BelledonneCommunications/bcg729"
+  "homepage": "https://github.com/BelledonneCommunications/bcg729",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
 }

--- a/versions/b-/bcg729.json
+++ b/versions/b-/bcg729.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "d515bbfce335039dc7edaa083d7ab334888f8254",
+      "version": "1.1.1",
+      "port-version": 2
+    },
+    {
       "git-tree": "16cf88e26d0bf3ebd469c722cd4df96c59556fb7",
       "version-string": "1.1.1",
       "port-version": 1

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -394,7 +394,7 @@
     },
     "bcg729": {
       "baseline": "1.1.1",
-      "port-version": 1
+      "port-version": 2
     },
     "bddisasm": {
       "baseline": "1.34.2",

